### PR TITLE
 ADD : teachers backend with integration

### PR DESCRIPTION
**Pull request by RaiDevX8**

### CODE REVIEW

File: client/.env.sample
Comment (Prithwi32) [inline client/.env.sample:1]: why necessary of / in url?

File: client/src/App.js
Comment (Prithwi32) [inline client/src/App.js:1]: use base url variable defined in .evn file

**Review by Prithwi32 — CHANGES_REQUESTED** [review]

File: client/src/App.js
Comment (RaiDevX8) [reply client/src/App.js:1]: Ok

File: client/.env.sample
Comment (RaiDevX8) [reply client/.env.sample:1]: done

**Review by RaiDevX8 — COMMENTED** [review]
sent another pr

**Comment by Prithwi32** [comment]
> sent another pr

It's fine..i changed and merged

**Pull request merged**